src/fluree/db/json_ld/commit.cljc

```diff
@@ -379,15 +416,15 @@
                                    new-commit*)
           [new-commit** jld-commit] (commit-data/commit-jsonld new-commit*)
           signed-commit (if did
-                          (cred/generate jld-commit private (:id did))
+                          (<? (cred/generate jld-commit private (:id did)))
```

Comment:
This fixed a bug where we we were writing out a stringified async channel instead of the actual signed commit, eg:
```
#object[clojure.core.async.impl.channels.ManyToManyChannel 0x66731af8 "clojure.core.async.impl.channels.ManyToManyChannel@66731af8"]%```